.pylintrc

```diff
@@ -1,7 +1,7 @@
 [MESSAGES CONTROL]
 disable=all
 enable=
-    C0121,C0200,C0209,C0321,C0325,C0415,
-    W0101,W0108,W0202,W0235,W0401,W0404,W0611,W0621,W0622,W612,W1309,
+    C0121,C0200,C0209,C0321,C0415,
```

Comment:
Why?

Reply:
it tells me in new pre-commit version that I have to delete it because it's currently are in newer versions